tooling/ipns/kubo_name.go

```diff
@@ -0,0 +1,147 @@
+/**
+ * This was copied from Kubo's `name` command
+ * TODO: discuss with the team about setting up a reusable abstraction somewhere.
```

Comment:
:thought_balloon: we've moved go-ipns to [`boxo/ipns`](https://github.com/ipfs/boxo/tree/main/ipns), my vote would be to move these utility functions there, and avoid duplicated code.

This cleanup could be a part of https://github.com/ipfs/specs/issues/376,  or something you  do before.